src/Core/Engine/Components/Transform.cpp

```
@@ -5,7 +5,7 @@
  *
  * @param translation Vector to add to the current position.
  */
-void COMPTransform::Move(const glm::vec3& translation)
+void COMPTransform::Move(const glm::vec3 translation)
```

Cppcheck (2.20.0)

[style] The function 'Move' is never used.

(unusedFunction)